bin/hdfs-mesos

```diff
@@ -1,5 +1,39 @@
 #!/bin/bash
 
+FRAMEWORK_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )/.."
+
+export MESOS_HDFS_DATA_DIR=${MESOS_HDFS_DATA_DIR:-"/tmp/hdfs/data"}
```

Comment:
This looks great to me, but let's update the README to let users know they need to set the environment variables instead of directly modifying mesos-site.xml. It might work it to include a note mentioning that one important variable to configure is MESOS_HDFS_ZKFC_HA_ZOOKEEPER_QUORUM since if you use localhost here, the executors will point to localhost and not to the zk quorum.